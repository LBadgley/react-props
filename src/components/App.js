import React from 'react';
import Dog from './Dog';
import Header from './Header';


export default function App() {
  console.log('that is my dog, maaaan');
  return (
    <>
      <Header />
      <Dog name="Ruby" age="2" weight="7.5 lbs"/>
    </>
  );
}
