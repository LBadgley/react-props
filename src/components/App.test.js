import React from 'react';
import { shallow } from 'enzyme';
import App from './App';

describe('app component test', () => {
  it('renders the app', () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });
});
