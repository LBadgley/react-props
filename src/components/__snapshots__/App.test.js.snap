// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app component test renders the app 1`] = `
<Color
  name="Ruby"
  rgb={
    Object {
      "blue": 88,
      "green": 61,
      "red": 168,
    }
  }
/>
`;
